docs/volto.md

```diff
@@ -0,0 +1,170 @@
+# Volto development
+
+`volto` is a React project that uses the [razzle](https://razzlejs.org/) build system.
+
+You can't make any useful headway in `volto` without a `plone` backend running.
+
+## Background
+
+We target `node` version 14 - which is the newest version of node that volto 13 supports. We use `yarn` 
+for dependency management etc.
+
+We have yarn workspaces registered in `package.json`. We have these so yarn fetches their dependencies along with
+those in package.json.
+
+We have forced some `resolutions` in package.json to prevent cross-version problems in various modules. React, for 
+example, doesn't like to have separate versions installed.
+
+There are two ways to work on volto; locally with node, or in a `docker-compose` app.
+
+## Addons
+
+The addons that volto loads are normally specified in the `package.json:addons` array. That can be
+overridden using the ADDONS env var.
+
+There are two aspects to using a volto addon. 
+
+First, the add on must be available as either a `node_module` or as a `mrs-developer` managed workspace in 
+this project. This step means the addon source will be present, and any dependencies it has will also be installed.
+
+Second, to use the addon, it must be present in the `package.json:addons` array. This will cause the volto to load 
```

Comment:
Ok, gotcha. I hadn't seen that this was possible and thought that there was some jiggery pokery going on in the Docker entry-point script to effectively update the `package.json`.

Have you got a link to the docs where they talk about this env variable overriding the `package.json:addons` array?

Reply:
Found it! (Neat life hack -- I used Google :) )

https://docs.voltocms.com/configuration/environmentvariables/#use-add-ons-via-the-addons-environment-variable

Reply:
Cunning ploy. Update looks good.